modules/cluster-logging-deploy-console.adoc

```diff
@@ -4,24 +4,25 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-deploy-console_{context}"]
-= Installing the {logging-title} using the web console
+= Installing the {clo} by using the web console
 
-[NOTE]
-====
-If you do not want to use the default Elasticsearch log store, you can remove the internal Elasticsearch `logStore` and Kibana `visualization` components from the `ClusterLogging` custom resource (CR). Removing these components is optional but saves resources.
-====
+You can install the {clo} by using the {product-title} web console.
+
+.Prerequisites
 
-include::snippets/logging-compatibility-snip.adoc[]
+* You have administrator permissions.
+* You have access to the {product-title} web console.
 
 .Procedure
 
 . In the {product-title} web console, click *Operators* -> *OperatorHub*.
+. Type `OpenShift Logging` in the *Filter by keyword* box.
 . Choose  *Red Hat OpenShift Logging* from the list of available Operators, and click *Install*.
 . Ensure that *A specific namespace on the cluster* is selected under *Installation mode*.
 . Ensure that *Operator recommended namespace* is *openshift-logging* under *Installed Namespace*.
 . Select *Enable operator recommended cluster monitoring on this namespace*.
 +
-This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object. You must select this option to ensure that cluster monitoring scrapes the `openshift-logging` namespace.
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the `Namespace` object. You must select this option to ensure that cluster monitoring scrapes the `openshift-logging` namespace.
```

Comment:
"Scrapes" seems a bit out of place when it comes to translation. Would "scans" be more suitable?

Reply:
It's generally referred to as "scraping" metrics 🙂 see https://prometheus.io/docs/prometheus/latest/getting_started/